README.md

```diff
@@ -30,40 +30,39 @@ curl -s https://raw.githubusercontent.com/faros-ai/faros-events-cli/$FAROS_CLI_V
 1. Pull the image:
 
 ```sh
-docker pull farosai/faros-events-cli:v0.5.2
+docker pull farosai/faros-events-cli:v0.5.3
 ```
 
 2. Run it:
 
 ```sh
-docker run farosai/faros-events-cli:v0.5.2 help
+docker run farosai/faros-events-cli:v0.5.3 help
 ```
 
 ### :book: Event Types
 
 An event type (e.g. `CI`, `CD`, `TestExecution`) corresponds to the step of your CI/CD or Test pipeline that you are instrumenting.
 
-- Use Continuous Integration (CI) events to instrument code build pipelines. For example, you can report the result of a successful code build & artifact release:
+- Use Continuous Integration (CI) events to instrument code build pipelines. For example, you can report the result of a successful code build:
 
 ```sh
 ./faros_event.sh CI -k "<faros_api_key>" \
-    --run "<run_source>://<run_organization>/<run_pipeline>/<run_id>" \
     --commit "<commit_source>://<commit_organization>/<commit_repository>/<commit_sha>" \
-    --artifact "<artifact_source>://<artifact_organization>/<artifact_repository>/<artifact_id>" \
```

Comment:
Artifact is now more of an advanced configuration option. I am not broadcasting it in the example events.